sci-libs/hdf5/hdf5-1.10.5.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+FORTRAN_NEEDED=fortran
+AUTOTOOLS_AUTORECONF=1
+
+inherit autotools-utils eutils fortran-2 flag-o-matic toolchain-funcs multilib versionator prefix
```

Comment:
autotools-utils is banned, eutils is not needed, multilib is not needed, versionator should be replaced

Reply:
ebuild breaks without `autotools-utils`.

Reply:
Migrate from autotools-utils. Just removing it from inherit is not sufficient of course.